src/AddIns/Uno.UI.MediaPlayer.Skia.Gtk/GTKMediaPlayer.events.cs

```diff
@@ -154,46 +103,28 @@ private void Initialized()
 		};
 
 		_ = Dispatcher.RunAsync(CoreDispatcherPriority.Normal, () =>
-{
-	Console.WriteLine("Set MediaPlayer");
-	_videoView.MediaPlayer = _mediaPlayer;
-
-	_ = Dispatcher.RunAsync(CoreDispatcherPriority.Normal, () =>
-	{
-
-		_videoView.Visible = true;
-		_videoView.MediaPlayer = _mediaPlayer;
-		_mediaPlayer.Stopped += (sender, e) =>
 		{
-			_videoView.Visible = false;
-		};
-		//Starts playing
-		var media = new LibVLCSharp.Shared.Media(
-			_libvlc,
-			new Uri("https://ia800201.us.archive.org/12/items/BigBuckBunny_328/BigBuckBunny_512kb.mp4")
-		);
-
-		media.Parse(MediaParseOptions.ParseNetwork);
-		_videoView.MediaPlayer.Media = media;
-
-		Console.WriteLine("Content _videoView on Dispatcher");
-		//_videoContainer.Content = new Border() { Background = new SolidColorBrush(Colors.Red) };
-		_videoContainer.Content = _videoView;
-		//_videoContainer.Height = 300;
-		//_videoContainer.Width = 450;
-		//Child = new Border() { Background = new SolidColorBrush(Colors.Red) };
-		Child = _videoContainer;
-
-
-
-
-		UpdateVideoStretch();
-		Console.WriteLine("Created player");
-	});
-});
-
-
+			Console.WriteLine("Set MediaPlayer");
+			_videoView.MediaPlayer = _mediaPlayer;
 
+			_ = Dispatcher.RunAsync(CoreDispatcherPriority.Normal, () =>
+			{
+				_videoView.Visible = true;
+				_videoView.MediaPlayer = _mediaPlayer;
+				_mediaPlayer.Stopped += (sender, e) =>
+				{
+					Console.WriteLine("MediaPlayer Stopped");
+					_videoView.Visible = false;
+				};
+
+				Console.WriteLine("Content _videoView on Dispatcher");
+				_videoContainer.Content = _videoView;
+				Child = _videoContainer;
+
+				UpdateVideoStretch();
+				Console.WriteLine("Created player");
+			});
+		});
 
 		SourceLoaded += OnHtmlSourceLoaded;
```

Comment:
Rename source loaded methods to remove html

Reply:
done